### PR TITLE
add support for HELMFILE_FILE_PATH envvar to set -f 

### PR DESCRIPTION
**Pull request by sstarcher**
https://github.com/helmfile/helmfile/discussions/1254

### CODE REVIEW

**Comment by sstarcher** [comment]
Any way to rekick off the tests as they don't look related to my change.  If not I can amend and push the commit back up.

**Comment by yxxhero** [comment]
@sstarcher please add some tests for this feature. thanks so much.

**Comment by sstarcher** [comment]
Any recommendations for where I should add a test for an if statement?  I don't see any existing tests for `global.go` unless you want me to add it at a much higher level?

**Comment by yxxhero** [comment]
@sstarcher please fix lint error.

**Comment by sstarcher** [comment]
@yxxhero not seeing any errors output in those logs.  Looks unrelated to me
```
run golangci-lint
  Running [/home/runner/golangci-lint-1.55.1-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  Error: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/helmfile/helmfile) (gci)
  Error: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/helmfile/helmfile) (gci)
  
  Error: issues found
  Ran golangci-lint in 5361ms
```

File: pkg/config/global_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/helmfile/helmfile/pkg/envvar"
+	"github.com/stretchr/testify/require"
```
Comment (yxxhero) [inline pkg/config/global_test.go:8]: update it like below:

"github.com/stretchr/testify/require"

"github.com/helmfile/helmfile/pkg/envvar"

File: pkg/config/global_test.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		},
+		{
+			opts:     GlobalOptions{File: "folderset"},
+			env:      "envset",
+			expected: "folderset",
+		},
+	}
+
+	for _, test := range tests {
+		os.Setenv(envvar.FilePath, test.env)
```
Comment (yxxhero) [inline pkg/config/global_test.go:41]: please unset the env when testing is done.

**Comment by sstarcher** [comment]
@yxxhero thanks for the fix.  Anything else needed to get this merged?

File: pkg/config/global_test.go (hunk 2 above)
Comment (yxxhero) [reply pkg/config/global_test.go:42]: @sstarcher

Comment (yxxhero) [reply pkg/config/global_test.go:42]: please see this and fix this. thanks so much. @sstarcher

**Review by yxxhero — APPROVED** [review]

**Comment by sstarcher** [comment]
Thanks

**Pull request merged**